docs/source/tips.rst

```diff
@@ -18,60 +18,3 @@ If you see that compilations run every time you run the same script, then the ca
 Please check that the directory is kept as is between multiple executions of the script.
 If your home directory is not suited to caching the kernels (e.g. in case that it uses NFS), change the kernel caching directory by setting the ``CUPY_CACHE_DIR`` environment variable to an appropriate path.
 See `CuPy Overview <https://docs-cupy.chainer.org/en/stable/overview.html>` for more details.
-
-
-mnist example does not converge in CPU mode on Mac OS X
--------------------------------------------------------
```

Comment:
I think (at least the first half of) this section is still needed to tell users how to install openblas on OS X.